deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -154,11 +154,25 @@ class ChatViewController: UITableViewController {
     }()
 
     private lazy var contextMenu: ContextMenuProvider = {
-        let copyItem = ContextMenuProvider.ContextMenuItem(
-        title: String.localized("global_menu_edit_copy_desktop"),
-        imageName: "doc.on.doc",
-        action: #selector(BaseMessageCell.messageCopy),
-        onPerform: { [weak self] indexPath in
+        let dcChat = dcContext.getChat(chatId: chatId)
+        let config = ContextMenuProvider()
+        if #available(iOS 13.0, *), dcChat.canSend {
+            let mainMenu = ContextMenuProvider.ContextMenuItem(submenuitems: [replyItem, forwardItem, infoItem, copyItem, replyPrivatelyItem, deleteItem])
+            config.setMenu([mainMenu, selectMoreItem])
+        } else if dcChat.canSend {
+            config.setMenu([forwardItem, infoItem, copyItem, replyPrivatelyItem, deleteItem, selectMoreItem])
```

Comment:
seems ios<13 does not have the "reply" function - what was the reason for that? and is it okay to allow replyPrivately? you tested on ios12, so probably fine - just asking :)

Reply:
I think we decided against it back in the days, because the horizontal space for items are limited. you need to press an arrow icn to see additional options, and repply was easily accessible with the swipe gesture